modules/tinymce/src/core/test/ts/browser/dom/ControlSelectionTest.ts

```diff
@@ -75,18 +75,24 @@ describe('browser.tinymce.core.dom.ControlSelectionTest', () => {
     const resizeHandle = await UiFinder.pWaitForVisible('Wait for resize handlers to show', editorBody, resizeSelector);
     const target = UiFinder.findIn(editorBody, targetSelector).getOrDie();
     Mouse.mouseDown(resizeHandle);
+    // a real user won't drag within a single frame, so we shouldn't either
+    await Waiter.pWait(17);
```

Comment:
So all these 17ms is that to simulate a frame wait. Why not use requestAnimationFrame? Who knows what the framerate is on the remote test machines. Or maybe we should have a Waiter.pWaitForFrame that we can tweak that does this or requestAnimationFrame.